esphome/components/deep_sleep/__init__.py

```diff
@@ -189,7 +203,11 @@ def _validate_ex1_wakeup_mode(value):
             cv.Optional(CONF_WAKEUP_PIN): cv.All(
                 cv.only_on_esp32,
                 pins.internal_gpio_input_pin_schema,
-                validate_pin_number,
+                validate_pin_number_esp32,
+            ),
+            cv.Optional(CONF_WAKEUP_PINS): cv.All(
+                cv.only_on(["libretiny", "bk72xx"]),
```

Comment:
```suggestion
                cv.only_on(["bk72xx"]),
```

pretty sure libretiny isn't needed here. need to check if it would match non bk72xx

Reply:
`PLATFORM_BK72XX`

Reply:
ok